models/DataObject/Service.php

```diff
@@ -1450,7 +1450,7 @@ public static function enrichLayoutDefinition(&$layout, $object = null, $context
      */
     public static function enrichLayoutPermissions(&$layout, $allowedView, $allowedEdit)
     {
-        if ($layout instanceof Model\DataObject\ClassDefinition\Data\Localizedfields) {
+        if ($layout instanceof Model\DataObject\ClassDefinition\Data\Localizedfields || $layout instanceof Model\DataObject\ClassDefinition\Data\Classificationstore && $layout->localized === true) {
```

Comment:
Please try to avoid extending the baseline. Would it be possible to do something like this instead ?
```
$layout = $layout->getObjectVars();
$layout['whateveryKey'] = $wahteverValue;
```

Reply:
i think this is worse because layout is by reference an object there. Also this wont work with protected properties.

I think the whole js handling of localized field permissions has to be adjusted either way or you add it to baseline

Localized Fields was probably in there because of this reason.

I agree that it is not pretty but if i start applying changes there it will be more than just a small bugfix.